keanu-python/tests/test_vertex.py

```diff
@@ -205,13 +207,20 @@ def test_you_can_set_value(vertex, expected_type, value, assert_vertex_value_equ
     assert_vertex_value_equals(vertex, expected_type, value)
 
 
+@pytest.mark.parametrize("vertex, expected_type", [(Gaussian(0., 1.), float), (UniformInt(0, 10), int),
+                                                   (Bernoulli(0.5), bool)])
+@pytest.mark.parametrize("value, assert_vertex_value_equals", [(4, assert_vertex_value_equals_scalar),
+                                                               (5., assert_vertex_value_equals_scalar),
+                                                               (True, assert_vertex_value_equals_scalar)])
+def test_you_can_set_scalar_value(vertex, expected_type, value, assert_vertex_value_equals):
+    test_you_can_set_value(vertex, expected_type, value, assert_vertex_value_equals)
```

Comment:
You can just call `assert_vertex_value_equals_scalar` directly here

Reply:
👍